packages/plugin/src/TablePlugin.ts

```diff
@@ -29,7 +29,7 @@ export interface TablePluginProps<S = unknown> {
   /**
    * The model for the table this plugin is associated with.
    */
-  model: IrisGridModel;
+  model: IrisGridTableModelTemplate;
```

Comment:
Mmm at some point wish we could clean up our GridModel types. Should all be defined as types/interfaces instead of classes.